services/core-web/src/components/modalContent/DownloadDocumentPackageModal.js

```diff
@@ -57,12 +57,22 @@ export const DownloadDocumentPackageModal = (props) => {
       />
       <br />
       <div className="right center-mobile padding-md--top">
+        <Popconfirm
+          placement="topRight"
+          title="Are you sure you want to cancel?"
+          onConfirm={props.closeModal}
+          okText="Yes"
+          cancelText="No"
+        >
+          <Button className="full-mobile">Cancel</Button>
+        </Popconfirm>
         <Button
```

Comment:
This should already be built into the base modal component - if you pass in the prop "isViewOnly={false}" it will add this to the modal

Reply:
disregard.